### ISSUE TITLE
VMagent process_cpu_seconds_total metric shows something strange

### ISSUE THREAD

**Issue by turbotankist** [VictoriaMetrics/VictoriaMetrics#3982]
### Describe the bug

I have 2 clusters with installed victoria-metrics-k8s-stack. And I added extra scape config for each vmagent to scrape metrics from another one.
And I receive an alert from one of them
`TooHighCPUUsage: More than 90% of CPU is used by "vmagents_external"("vmagent1_url443)") during the last 5m - CRITICAL`

the alert metric show this
![image](https://user-images.githubusercontent.com/9462236/226373336-19675f2e-23c4-4d85-b5dd-3670cb3aa40b.png)
So process_cpu_seconds_total much more than 1.
I've checked metric `process_cpu_cores_available` - it has always value - 2
But process_cpu_seconds_total shows 
![image](https://user-images.githubusercontent.com/9462236/226374108-0826e2c8-f2bb-4dd7-bf8d-9075e53638ce.png)

I have no Idea what is `process_cpu_seconds_total` mean. 
Becouse the metric rate(container_cpu_usage_seconds_total{}) of the vmagent have no any issues 
![image](https://user-images.githubusercontent.com/9462236/226374930-0d8eaba8-e986-4848-bfe7-d801efe3d8fc.png)

throttling is about 0.
Why I receive an alert  TooHighCPUUsage for this pod?

### To Reproduce

metric `rate(process_cpu_seconds_total[5m]) / process_cpu_cores_available > 0.9` has strange result 

### Version

helm
  - name: victoria-metrics-k8s-stack
    version: "0.14.6"
 image_tag: v1.86.2

**Comment by hagen1778** [comment]
Hello!

What Operating System do you use for running VictoriaMetrics?

> I've checked metric process_cpu_cores_available - it has always value - 2
But process_cpu_seconds_total shows

the screen is cropped a bit - what Y axis shows on it?

**Comment by turbotankist** [comment]
> What Operating System do you use for running VictoriaMetrics?
We run it on EKS

I think I understand the reason.
We have on one side 2 replicas of vmagent. 
In another side - vmagent gets scrape of the first vmagent via ingress. And evrytime it get metrics from random vmagent replicas. 
So the result is a trash. 
Yeah -I've checked delta() of metrics - we have negative too. 
![image](https://user-images.githubusercontent.com/9462236/226606070-543d6b13-0504-402e-b381-4f1f19fe84aa.png)
We made it only to catch if first cluster will unavailable. But default victoriametrics alerts started to send notification.

Thank you for your time.  I'll close it
